packages/okta-react-native/ios/OktaSdkBridge.swift

```diff
@@ -0,0 +1,373 @@
+/*
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Foundation
+import OktaOidc
+
+@objc(OktaSdkBridge)
```

Comment:
OktaSdkBridge is too common class name. In fact it is OIDC bridge between js and native code
I would name it something like: RNOktaOidcBridge

Reply:
Will do this later since that might be a breaking change and i would want everything else to work first